src/packages/excalidraw/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Please add the latest change on the top under the correct section.
 
 ### Features
 
+- Support updating appState in [`updateScene`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L282) API.
```

Comment:
```suggestion
- Support updating any props in appState in [`updateScene`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L282) API.
```

Reply:
We don't take `appState` as props so won't it be confusing here if `appState` props is mentioned here?

Reply:
Props in this case is meant as properties of `appState`. The previous statement isn't quite correct because we were already support updating `appState` (just partial).

Reply:
In that case, we can use `properties` / `attributes` instead of `props` to be more clear, as `props` confused me when I read it for the first time.

Reply:
:+1: